Private Key Dashboard Instructions (#14)
oasislabs/docs: quickstart.md

2. `cd ../app`
3. Install app dependencies using `npm install` (or `yarn install` if that's more your style)
4. In a separate terminal, run the local chain using `oasis-chain`

> You can now test on the local chain using `oasis test --profile local`. Note that the `app` tests will run when in a subdirectory of `app`. You will see the following output,

Reviewer: original was correct

```
 PASS  test/service.spec.ts (5.965s)
  HelloWorld Test
    ✓ deployed (2ms)
    ✓ known greeting (409ms)
    ✓ insert new greeting in Samoan (400ms)

Test Suites: 1 passed, 1 total
Tests:       3 passed, 3 total
```

## Integration Test Using Devnet 2.0

> 1. Login to the [Oasis developer dashboard](https://dashboard.oasiscloud-staging.net) and visit `My Account`. Navigate to the `Payments` tab

Reviewer: staging?

2. `cd ../app`
3. Install app dependencies using `npm install` (or `yarn install` if that's more your style)
4. In a separate terminal, run the local chain using `oasis-chain`
5. Login to the [Oasis developer dashboard](https://dashboard.oasiscloud-staging.net) and visit `My Account`. Navigate to the `Payments` tab.
_Note:_ If you are a first-time user, you may be prompted to generate an Oasis-managed wallet, which you should do. Your wallet is what serves as your identity and enables you to interact with Oasis services.
6. Make sure you are in a secure location, and then `Click to reveal`. This will reveal your _private key_, which is used to access your wallet should never be shared with anyone.

> 7. Open `test/service.spec.ts`. Replace `<your private key>` with your private key.

Reviewer: nope. add it to the config file. better still: make a cli command that opens up a browser and pulls out the key from the webpage or something. or make an `oasis config profile.local private_key xxxx` or something

4. In a separate terminal, run the local chain using `oasis-chain`

You can now test on the local chain using `oasis test --profile local`. Note that the `app` tests will run when in a subdirectory of `app`. You will see the following output,

```
 PASS  test/service.spec.ts (5.965s)
  HelloWorld Test
    ✓ deployed (2ms)
    ✓ known greeting (409ms)
    ✓ insert new greeting in Samoan (400ms)

Test Suites: 1 passed, 1 total
Tests:       3 passed, 3 total
```

## Integration Test Using Devnet 2.0

1. Login to the [Oasis developer dashboard](https://dashboard.oasiscloud-staging.net) and visit `My Account`. Navigate to the `Payments` tab
_Note:_ If you are a first-time user, you may be prompted to generate an Oasis-managed wallet, which you should do. Your wallet is what serves as your identity and enables you to interact with Oasis services
2. Make sure you are in a secure location, and then `Click to reveal`. This will reveal your _private key_, which is used to access your wallet should never be shared with anyone
3. Set your private key for use by the client using `oasis config profile.default private_key[=]"private key"`

> You can now test on Devnet 2.0, similar to the way you tested on the local chain but this time using `oasis test`.

Reviewer: ```suggestion
You can now deploy to Devnet 2.0, using `oasis deploy`.
```

Testing on devnet? Paying actual money forever to test? Waiting for the entire thing to deploy to test? Pass.

We need a migrations-like script that the cli runs. `npm run-script deploy`, more like
